mesonbuild/dependencies/misc.py

```diff
@@ -614,7 +614,9 @@ def __init__(self, environment, kwargs):
                     self.is_found = True
                     self.compile_args = fwdep.get_compile_args()
                     self.link_args = fwdep.get_link_args()
-                    self.version = '2'  # FIXME
+                    # FIXME: Test on macOS
+                    #self.version = self.get_pcap_lib_version()
```

Comment:
__[flake8]__
* [E265] block comment should start with '# '

<!-- Comment by [SideCI](https://sideci.com) -->